stac_fastapi/pgstac/stac_fastapi/pgstac/core.py

```diff
@@ -23,6 +23,18 @@
 NumType = Union[float, int]
 
 
+def _extended_links(
+    original_links: Optional[List[Dict[str, Any]]], new_links: List[Dict[str, Any]]
+) -> List[Dict[str, Any]]:
+    """Extend links with new links."""
+    if original_links is None:
+        return new_links
+    new_link_rels = set([link["rel"] for link in new_links])
+    return new_links + [
+        link for link in original_links if link["rel"] not in new_link_rels
+    ]
+
+
```

Comment:
I'm wondering if this shouldn't exist in the `get_links` method under logic triggered by the `extra_links` argument, here: https://github.com/stac-utils/stac-fastapi/blob/b26a12ae210ac65fed2fe9344a13c1357281dce5/stac_fastapi/pgstac/stac_fastapi/pgstac/models/links.py#L94-L98

I believe the intent of that argument is to allow extensions of the generated links; it appears to not be used in the updated portions here

Reply:
Good call, changed in 4c37355